models/migrations/v1_21/v267.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_21 //nolint
+
+import (
+	"xorm.io/xorm"
+)
+
+// UpdateActionsRefIndex updates the index of actions ref field
+func UpdateActionsRefIndex(x *xorm.Engine) error {
+	type ActionRun struct {
+		Ref string `xorm:"index"` // the ref of the run
```

Comment:
The what of the run?

Reply:
Is that supposed the git action because of which the run is run?
In that case, what about something like
```suggestion
		Ref string `xorm:"index"` // the commit/tag/… causing the run
```
?